cmd/internal/cli/key_newpair.go

```diff
@@ -6,34 +6,194 @@
 package cli
 
 import (
+	"errors"
+	"fmt"
 	"os"
 
+	"github.com/sylabs/singularity/internal/pkg/util/interactive"
+
+	"github.com/sylabs/singularity/pkg/cmdline"
+	"github.com/sylabs/singularity/pkg/sypgp"
+
 	"github.com/spf13/cobra"
 	"github.com/sylabs/singularity/docs"
 	scs "github.com/sylabs/singularity/internal/pkg/remote"
 	"github.com/sylabs/singularity/internal/pkg/sylog"
-	"github.com/sylabs/singularity/pkg/sypgp"
 )
 
-// KeyNewPairCmd is 'singularity key newpair' and generate a new OpenPGP key pair
-var KeyNewPairCmd = &cobra.Command{
-	Args:                  cobra.ExactArgs(0),
-	DisableFlagsInUseLine: true,
-	PreRun:                sylabsToken,
-	Run: func(cmd *cobra.Command, args []string) {
-		keyring := sypgp.NewHandle("")
-		handleKeyNewPairEndpoint()
-
-		if _, err := keyring.GenKeyPair(keyServerURI, authToken, keyNewpairBitLength); err != nil {
-			sylog.Errorf("creating newpair failed: %v", err)
-			os.Exit(2)
+var (
+	keyNewPairName     string
+	KeyNewPairNameFlag = &cmdline.Flag{
+		ID:           "KeyNewPairNameFlag",
+		Value:        &keyNewPairName,
+		DefaultValue: "",
+		Name:         "name",
+		ShortHand:    "N",
+		Usage:        "keys owner name",
```

Comment:
```suggestion
		Usage:        "key owner name",
```

Although, `keys` (with the `s`) would be correct, because theres two keys. But `key` sounds better imo.